doc/user-guide/push-notifications/Push-notifications.md

```diff
@@ -0,0 +1,217 @@
+## How to set up Push Notifications
+
+Push notifications are the bread and butter of the modern mobile experience, and MongooseIM has
+support for them. When used together with [MongoosePush][], you get out-of-the-box push
+notifications for FCM (**F**irebase **C**loud **M**essaging) and APNS (**A**pple **P**ush
+**N**otification **S**ervice) providers. And it's easy to extend it to any other protocols of your
+choice.
+
+You might also want to read about the push notification's [client side configuration][client-side].
```

Comment:
I believe it's my last complaint, but this file is only available through this one link, and when you enter it, you are nowhere in the docs tree (it is not listed in the files structure on the left). I don't have any idea how to fix this problem.
Perhaps a link in line 105 would at least make the chance of finding this file a little higher.